tests/bits/fe_q_constraints.output

```diff
@@ -1,16 +1,16 @@
 
 DEAL::---------------------------------------------------------
-DEAL::P-Order: 1  Number of degrees of freedom: 24070
-DEAL::L2-Norm of test function 0.00307079
+DEAL::P-Order: 1  Number of degrees of freedom: 23529
+DEAL::L2-Norm of test function 0.163112
```

Comment:
Do you know what causes such a large change?

Reply:
I am not particularly sure about the impact of this PR on this test.

I assume that this is to the nature of the test scenario: We distribute random refinement indicators on the whole mesh. It might happen that with this PR, we now exclude crucial parts of the mesh that have a big impact on the global error. I think this will be the main cause for this, since refinement happens arbitrarily.
https://github.com/dealii/dealii/blob/0ad551ed25348288e5d7917a40c284a14bc74c5c/tests/bits/fe_q_constraints.cc#L142-L149

Reply:
Ah, good point. I should have just looked at the test. Thanks!